bin/hdfcoinc/pycbc_make_coinc_search_workflow

```diff
@@ -361,6 +361,10 @@ wf.make_segments_plot(workflow, [sci_ok_seg_file],
 wf.make_gating_plot(workflow, full_insps, rdir['analysis_time/gating'],
                     tags=['full_data'])
 
+if workflow.cp.has_option_tags('workflow-matchedfilter',
+                                   'plot-throughput', tags=[tag]):
```

Comment:
Hi @hfh12a. Many thanks for putting this patch together, it will be *really* useful to have these plots!

I just realised that my recent runs in fact did *not* have these plots (which I was looking forward to seeing). I tracked the reason back to these lines. Specifically the plot-throughput stage is optional, and currently not enabled in our ER10 ini files. Is there a specific reason to make these plots optional? It looks like all of the information will be written by default, so why not just make these plots run automatically?